Mathlib/Order/OrderType.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) _
```

Comment:
Reminder to fill this out; you can copy the header from another file and just modify the necessary fields.